karate-tests/src/test/java/patients/healthcareWorker/patientLifecycle.feature

```diff
@@ -0,0 +1,15 @@
+@wip
```

Comment:
Are we merging in WIP tests? Are you waiting for further work on the NHS number allocation to be completed?

Reply:
This one actually related to a demo I was preparing for the mocks. I can just remove it.

Reply:
I've removed this, the mocks definition and the mock runner java class. They'll all go back in when I raise a more specific PR related to mocks.